packages/pds/src/sequencer/events.ts

```diff
@@ -22,11 +22,15 @@ export const formatSeqCommit = async (
   const blobs = new CidSet()
   let carSlice: Uint8Array
 
+  const blocksToSend = new BlockMap()
+  blocksToSend.addMap(commitData.newBlocks)
+  blocksToSend.addMap(commitData.relevantBlocks)
```

Comment:
Can we omit `commitData.newBlocks` here?  If I understood correctly, the way things are currently written every "new" block is "relevant".  Or perhaps this is just a way to be defensive around missing any blocks.

Reply:
This will de-duplicate fwiw so we won't actually be sending along double blocks.

The way it's currently written "new blocks" are not all necessarily "relevant". Maybe a better word for "relevant blocks" would be "proof blocks". 

As an example take this edge case of a node split from the MST tests: https://github.com/bluesky-social/atproto/blob/main/packages/repo/tests/mst.test.ts#L328

The newly added record (`f`) has very few blocks that are relevant to it's proof (just the root of the tree). However the new MST has many new blocks in it to compensate for the split